resources/istio/charts/mixer/templates/config.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     request.scheme:
       valueType: STRING
     request.total_size:
-      valueType: INT64
+          valueType: INT64
```

Comment:
It will work, because first line in structure can have arbitrary indentation, and following must adapt. See more here: http://yaml.org/spec/1.2/spec.html#id2793979

Reply:
Yes, you are right. My bad.